src/app/screens/books/styles.js

```diff
@@ -0,0 +1,27 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'space-around'
+  },
+  book: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    margin: 10
+  },
+  bookImage: {
+    borderRadius: 50,
+    height: 40,
+    width: 40
```

Comment:
Extract this `40` as a const within this file, so this relation keeps attached to a single source :D